### DISCUSSION TITLE
Make TAB to go directly to the menu

### DISCUSSION THREAD

**Discussion by nunojsa** [marlonrichert/zsh-autocomplete#703] [Q&A]
Hi there,

So having the following key bindings:

```
bindkey '\t' menu-select "$terminfo[kcbt]" menu-select
bindkey -M menuselect '\t' menu-complete "$terminfo[kcbt]" reverse-menu-complete
```

work as expected. The problem comes when `insert-unambiguous` is set to yes. In that case, I would expect the first tab to auto complete only the common prefix. But what happens is that it goes directly to the menu complete. So the question is, is there any way to have the first tab to complete the common prefix and second one to go directly to the menu?

**Comment by marlonrichert** [comment]
Works for me with the latest version. Please update and try again.

**Comment by alexwforsythe** [comment]
Try adding all 4 of the configs from https://github.com/marlonrichert/zsh-autocomplete/discussions/664 (including the correction). That's worked for me
